pyteal/ast/txn.py

```diff
@@ -85,6 +85,7 @@ class TxnField(Enum):
     local_num_uints = (54, "LocalNumUint", TealType.uint64, 3)
     local_num_byte_slices = (55, "LocalNumByteSlice", TealType.uint64, 3)
     extra_program_pages = (56, "ExtraProgramPages", TealType.uint64, 4)
+    nonparticipation = (57, "Nonparticipation", TealType.uint64, 2)
```

Comment:
Unfortunately this isn't documented anywhere, but the last element of this tuple is the minimum TEAL version required to access that field, so it should be 5.